onadata/libs/templates/account_lockout/lockout_start.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans with lockout_time=lockout_time username=username %}
+Hi {{ username }},
+
+You have been locked out from your Ona account because of too many wrong passwords.
```

Comment:
I would go more with "You have entered too many wrong passwords" @rowo What are your thoughts?

Reply:
Is this an email in addition to messaging in the UI where the attempts happened?

Reply:
When I've seen this it's not clear that it's only a password problem — it's a login attempt (user name + password) problem. See "Warn Users Before Locking Their Account" section on https://uxplanet.org/most-common-log-in-problems-and-solutions-163a6209c0eb which uses the language, "For security reasons, after 9 more failed login attempts you'll have to wait 30 minutes before trying again." The specific number of failed attempts is useful.

Reply:
>Is this an email in addition to messaging in the UI where the attempts happened?
Yes it is.
>When I've seen this it's not clear that it's only a password problem — it's a login attempt (user name + password) problem. See "Warn Users Before Locking Their Account" section on https://uxplanet.org/most-common-log-in-problems-and-solutions-163a6209c0eb which uses the language, "For security reasons, after 9 more failed login attempts you'll have to wait 30 minutes before trying again." The specific number of failed attempts is useful.
Just updated the error messages on the UI. The warning starts upon first invalid login, and the number of remaining attempts in the warning decrease with every subsequent login attempt.